content/docs/user-guide/gems/reference/terrain/layer_spawner.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Terrain Layer Spawner component
+linktitle: LayerSpawner
+description: ' Open 3D Engine (O3DE) Terrain Layer Spawner reference. '
+weight: 100
+---
+
+The **Terrain Layer Spawner** component spawns a terrain layer within given bounds, and enables the ordering of multiple layers.  
+The priority is controlled firstly by assigning a layer, either **Foregound**(Highest priority) or **Background**, then by using the **Priority** setting, with a higher number being a higher priority.  
+The dimensions of the layer can be configured by modifying the [Box component](/docs/user-guide/components/reference/shape/box-shape) on the same entity.
+The **Box component** is required for the **Layer Spawner** to operate.
+
+## Provider ##
+
+[Terrain Gem](/docs/user-guide/gems/reference/terrain)
+
+## Properties ##
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Layer Priority** | The priority of the spawner, Foreground has a higher priority than Background. | Foreground or Background | `Foreground` |
```

Comment:
```suggestion
| **Layer Priority** | The priority of the spawner, Foreground, has a higher priority than Background. | Foreground or Background | `Foreground` |
```
Make sure that the punctuation addition clarifies the intended meaning.

Reply:
> 
> 
> Make sure that the punctuation addition clarifies the intended meaning.

I think this should be two sentences based on context from the interface.

```suggestion
| **Layer Priority** | The priority of the spawner. Foreground has a higher priority than Background. | Foreground or Background | `Foreground` |
```

Reply:
Oh I see now -- yea I read that wrong!

Reply:
Yes, the two sentence form is better, a colon would have worked as well. Changed.